README.md

```diff
@@ -46,19 +46,19 @@ from __future__ import print_function
 
 import squareconnect
 from squareconnect.rest import ApiException
-from squareconnect.apis.location_api import LocationApi
+from squareconnect.apis.locations_api import LocationsApi
 
 # setup authorization
 squareconnect.configuration.access_token = 'YOUR_ACCESS_TOKEN'
 # create an instance of the Location API class
-api_instance = LocationApi()
+api_instance = LocationsApi()
 
 try:
     # ListLocations
     api_response = api_instance.list_locations()
     print (api_response.locations)
 except ApiException as e:
-    print ('Exception when calling LocationApi->list_locations: %s\n' % e)
```

Comment:
The syntax in question is existing, and IIRC the same throughout the python api. While it does conflict with the PEP8 recommendations, changing it would be something that should be done in a different pull request.

This change LGTM